CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -815,6 +817,23 @@ function PlayerHospital:tickEarthquake(stage)
   end
 end
 
+--! Give advice that a patient is waiting for the player to build a GP's office
+-- Called when a patient has passed reception and is waiting for a room to be built.
+-- Each piece of advice is only given once per month
+function PlayerHospital:adviseNoGpsOffice()
+  if self:countStaffOfCategory("Doctor", 1) > 0 then -- Doctor without a room
+    if not self.adviser_data.gps_office_1 then
+      self.world.ui.adviser:say(_A.warnings.no_gps_office_2)
+      self.adviser_data.gps_office_1 = true
```

Comment:
Coupling a `_1` variable to a `_2` variable elsewhere??? Who are you trying to confuse here?

The better option in any case is not to have these number suffixes. If you must, then please make it trivial to understand how they are related, eg by using the same number pairwise. Humans are bad at these guessing games.